Makefile

```diff
@@ -23,7 +23,7 @@ dev.up:  # Start Blockstore container
 	docker-compose --project-name "blockstore${PYTHON_VERSION}" -f "docker-compose-${PYTHON_VERSION}.yml" up -d
 
 dev.provision:  # Provision Blockstore service
-	docker exec -t edx.devstack.mysql /bin/bash -c 'mysql -uroot <<< "create database if not exists blockstore_db;"'
+	docker exec -t edx.devstack.mysql57 /bin/bash -c 'mysql -uroot <<< "create database if not exists blockstore_db;"'
 	docker exec -t ${CONTAINER_NAME} /bin/bash -c 'source ~/.bashrc && make requirements && make migrate'
 
 stop:  # Stop Blockstore container
```

Comment:
Can you add the two missing `${PYTHON_VERSION}` to the `docker-compose` command in the next line?